src/app/services/api.ts

```diff
@@ -24,7 +24,7 @@ export const api = createApi({
         'Levels',
         'Challenges',
         'Idle_Members',
-        'User_Standup',
```

Comment:
Why are you removed?

Reply:
Because I changed it to getProgress a common provider tag